### PR TITLE
Add spatial readme template

### LINKED ISSUE #204: Update Readme for Spatial Data
### Context

We will be adding spatial data to downloads soon. We need to update the readme to account for this.

### PR DESCRIPTION
**Pull request by allyhawkins**
## Issue Number
Closes #204 

## Purpose/Implementation Notes
Here I have added in a new template readme that will specifically be for spatial downloads. I named the file `spatial_readme_template.md` and stored it in the same folder as the original `readme_template.md`. 

## Types of changes

For the most part I took what was in the original readme and copied that over, keeping the same introduction, contact, citing, and terms of use. The main difference between the spatial readme and the readme used for all other downloads is the contents section which now includes the contents of a spatial folder, rather than the single-cell gene expression contents. Here I mostly took what we have in the [spatial docs](https://github.com/AlexsLemonade/scpca-docs/blob/main/docs/download_files.md#spatial-transcriptomics-libraries), listing the contents of the spatial folder and then linking to the docs for more information. I also included a link to the processing information for spatial libraries as well.  

Some questions I have: 

- Is the file name and location okay? @davidsmejia @arkid15r 
- Are there any other contents that are missing related to spatial files that we should include? 

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/scpca_portal/config/spatial_readme_template.md
Comment (jashapiro) [inline api/scpca_portal/config/spatial_readme_template.md:22]: Do we want to link directly to the spatial section of the docs? 

Which leads me to the question: why is the spatial section not showing up in https://scpca.readthedocs.io/en/latest/download_files.html ? Some of the spatial content is there for processing, but the section [here](https://github.com/AlexsLemonade/scpca-docs/blob/main/docs/download_files.md#spatial-transcriptomics-libraries) is not showing up on readthedocs.io

**Review by jashapiro — APPROVED** [review]
This looks good to me, but it raised a separate question about readthedocs.io, which doesn't seem to be showing the latest revisions on GitHub. Not quite sure what is up with that.

File: api/scpca_portal/config/spatial_readme_template.md
Comment (allyhawkins) [reply api/scpca_portal/config/spatial_readme_template.md:22]: Okay one option is to add it now and it will be a broken link that should work when things are deployed... or we could hold off and update it later when the docs are updated...?

**Comment by arkid15r** [comment]
> Is the file name and location okay? @davidsmejia @arkid15r

Don't worry about this, I'll rename/relocate it as needed after a merge. The content is important though :)

**Pull request merged**
